pkg/monitor/cluster/clustersync.go

```diff
@@ -0,0 +1,67 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitClusterSync(ctx context.Context) error {
+	clusterSync, err := mon.hiveClusterManager.GetClusterSync(ctx, mon.doc)
+	if err != nil {
+		return err
+	}
+	if clusterSync != nil {
+		if clusterSync.Status.SyncSets != nil {
+			for _, s := range clusterSync.Status.SyncSets {
+				mon.emitGauge("hive.clustersync", 1, map[string]string{
```

Comment:
question: Do we want to change what "value" we emit here based on the success/failure state of the syncset? For example, return `1` for Successful syncsets and `0` for failed syncsets? 

This might make it easier for us to, for example, build downstream dashboards or alerts based off of this metric.

Reply:
Value `1` is utilized to transmit data whenever SyncSets succeed or fail. The dashboard has been designed accordingly to display the number of successful and failed SyncSets both.